examples/light-switch-app/silabs/include/LightSwitchMgr.h

```diff
@@ -39,20 +49,69 @@ class LightSwitchMgr
         chip::EventId event;
     };
 
+    static constexpr uint8_t stepSize = 1;
+    static constexpr DataModel::Nullable<uint16_t> transitionTime = 0;
+    static constexpr chip::BitMask<OptionsBitmap> optionsMask = 0;
+    static constexpr chip::BitMask<OptionsBitmap> optionsOverride = 0;
```

Comment:
These represent a Step command, we should use the step command instead

Reply:
Used step command